client/a8c-for-agencies/sections/marketplace/products-overview-v2/woopayments-product-card/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { Button } from '@wordpress/components';
+import { check } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useMemo, useState } from 'react';
+import WooPaymentsLogo from 'calypso/assets/images/a8c-for-agencies/product-logos/woopayments.svg';
+import { useURLQueryParams } from 'calypso/jetpack-cloud/sections/partner-portal/hooks';
+import { LICENSE_INFO_MODAL_ID } from 'calypso/jetpack-cloud/sections/partner-portal/lib';
+import LicenseLightbox from 'calypso/jetpack-cloud/sections/partner-portal/license-lightbox';
+import LicenseLightboxLink from 'calypso/jetpack-cloud/sections/partner-portal/license-lightbox-link';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { APIProductFamilyProduct } from 'calypso/state/partner-portal/types';
+
+import './style.scss';
+
+interface Props {
+	asReferral?: boolean;
+	product: APIProductFamilyProduct;
+	isSelected: boolean;
+	onSelectProduct: ( value: APIProductFamilyProduct ) => void | null;
+	suggestedProduct?: string | null;
+	hideDiscount?: boolean;
+	quantity?: number;
+}
+
+export default function WooPaymentsProductCard( {
+	asReferral,
+	product,
+	isSelected,
+	onSelectProduct,
+	quantity,
+}: Props ) {
+	const { setParams, resetParams, getParamValue } = useURLQueryParams();
+	const modalParamValue = getParamValue( LICENSE_INFO_MODAL_ID );
+
+	const [ showLightbox, setShowLightbox ] = useState( modalParamValue === product.slug );
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const onSelect = useCallback( () => {
+		onSelectProduct?.( product );
+	}, [ onSelectProduct, product ] );
+
+	const onKeyDown = useCallback(
+		( e: any ) => {
```

Comment:
What's the reason to use `any in this case?_

Reply:
Good catch! This was a copy and paste, and I forgot to include the correct type here.